### ISSUE TITLE
bug: qs is broken on main branch

### ISSUE THREAD

**Issue by clubanderson** [kubestellar/kubestellar#767]
### Describe the bug

qs is not running through docs-ecutable properly

https://github.com/kubestellar/kubestellar/actions/runs/5482068175/jobs/9987030751

```
Execution follows
+ /home/runner/work/kubestellar/kubestellar/docs/scripts/generated_script.sh
++ bash /dev/fd/63 --kubestellar-version latest
+++ curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/main/bootstrap/bootstrap-kubestellar.sh
< KubeStellar bootstrap started >----------------
< Ensure kcp is installed >----------------------
kcp not found in the PATH.
Downloading kcp+plugins v0.11.0 linux/amd64...
Installing kcp+plugins into '/home/runner/work/kubestellar/kubestellar/kcp'...
Add kcp folder to the PATH: export PATH="/home/runner/work/kubestellar/kubestellar/kcp/bin:$PATH"
< Ensure kcp is running >-----------------------
Running kcp... logfile=/home/runner/work/kubestellar/kubestellar/kcp_log.txt
Waiting for kcp to be ready... it may take a while
kcp version v0.11.0 ... ok
Export KUBECONFIG environment variable: export KUBECONFIG="/home/runner/work/kubestellar/kubestellar/.kcp/admin.kubeconfig"
< Ensure KubeStellar is installed >-------------
KubeStellar not found in the PATH.
Downloading KubeStellar Search code, repositories, users, issues, pull requests... linux/amd64...
curl: (3) URL using bad/illegal format or missing URL
make: *** [Makefile:203: docs-ecutable] Error 3
Error: Process completed with exit code 2.
```

### Steps To Reproduce

1. start docs-ecutable for QS on any branch
2. observe failure of bootstrap

### Expected Behavior

successful runs should occur on all branches

**Comment by francostellari** [comment]
@clubanderson this issue should be fixed by PR #761

**Comment by clubanderson** [comment]
fixed - Thank you @francostellari
